pkg/kubectl/cmd/get.go

```diff
@@ -69,6 +72,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			err := RunGet(f, out, cmd, args)
 			cmdutil.CheckErr(err)
 		},
+		ValidArgs: validArgs,
```

Comment:
Maybe, if the primary use is suggesting completions. I think this is fine, just the wording of the [comment](https://github.com/spf13/cobra/blob/master/command.go#L47) on ValidArgs made me think this was expected to be a complete list.